### DISCUSSION TITLE
Append Link custom display link

### DISCUSSION THREAD

**Discussion by prattatx** [chhoumann/quickadd#640] [Ideas]
My workflow is typing the title of a file in a daily note (say, a meeting). Then I highlight and have quickadd apply a template. It creates the new file and inserts the link to that file in the previous note. I have quickadd drop unique codes on top of the link to the new note. 

It might be nice to have the original text as a "custom text" of the link, and the new note with the unique codes appended. 

so that we have 

original file: Meeting with Mark
- highlight that and quick add meeting note template
- file created with ```[[20240101 Meeting with Mark]]```
- Link in original file is ```[[20240101 Meeting with Mark|Meeting with Mark]]```

**Comment by snovis** [comment]
For what it's worth, I would love to do something similar.  I want to be able to select a block of text, create a new zettlekasten note from that selection - but have the selection replaced with a link to the new file

Confirmation Bias becomes [[202405051124 - Confirmation Bias|Confirmation Bias]] in the active document.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

Still not built: when QuickAdd replaces your selection with a link to the new note, the link is generated without an alias, so there is currently no way to keep "Meeting with Mark" as the display text. It fits cleanly into the link options that Template and Capture choices have grown since (placement, link vs. embed), so I'm now tracking it in #1479. No promises on timing.

@snovis your zettelkasten variant is the same feature - one display-text option would cover both.

Closing here so the conversation continues on the issue.
